bittensor/_synapse/text_causallmnext_impl.py

```diff
@@ -18,6 +18,7 @@
 import bittensor
 import torch
 from .synapse_impl import Synapse
+from bittensor.utils.tokenizer_utils import compact_topk_token_phrases, unravel_topk_token_phrases
```

Comment:
compact is the encoding functions while unravel is the decoding function, correct?

Reply:
Yes, compact is encoding for the wire, and unravel is decoding from the wire.